client/Dockerfile

```diff
@@ -13,5 +13,14 @@ RUN npm run build
 FROM nginx:stable-alpine
```

Comment:
For the sake of reproducibility, we prefer using specific version. Dependabot is configured to let us know when a new version is available.

Reply:
Done! And did same for the Node base image, too.